build/build-image/Dockerfile

```diff
@@ -108,7 +111,7 @@ RUN echo "export EDITOR=nano" >> /root/.bashrc
 RUN wget -nv -O terraform.zip https://releases.hashicorp.com/terraform/1.0.8/terraform_1.0.8_linux_amd64.zip && unzip ./terraform.zip && mv terraform /usr/local/bin/
 
 # code generation scripts
-COPY *.sh /root/
+COPY /scripts/build-image/*.sh /root/
```

Comment:
🤔 

1. What's the impetus to move all the scripts into a seperate folder away from the Dockerfile? I figured having them next to each other made discovery easier? (now I need to go to a different folder, match the name of this folder to it's subfolder, and then work from there - which is a tad tricky).
2. If we do have a ./scripts/ folder, rather than having it sit as a new top level folder, it should probably sit within `/build` since that houses all our build scripts and related.

Reply:
I move them to the top level scripts folder because the docker file needs node package in example folder. I can't put the node package in build folder because the build folder is excluded from dockerfile at the example folder level. Right now I changed the make file in build/includes to cd to the agones repo to get a copy of node package, thus I need to move the script files in build folder to top level to avoid it excluded from docker. Let me know if you have better locations to put them. Thanks!

Reply:
> I move them to the top level scripts folder because the docker file needs node package in example folder

I still feel like we should be able to download this from somewhere, and then this goes away?

If the scripts were shared between images, I would get having some sort of `common` directory that each image could pull from, but we don't really do that anywhere - so this aspect is still confusing me.

It's already a bit complicated that you have to match `build/build-sdk-images/<lang>` to `sdk/<lang>` appropriately, but now we're adding another layer to that of `build/scripts/<lang>` - and I'm still not sure of the value add in doing so.